samples/app/src/androidTest/java/com/sourcepoint/app/v6/SpClientMock.kt

```diff
@@ -1,32 +0,0 @@
-package com.sourcepoint.app.v6
-
-import android.view.View
-import com.sourcepoint.cmplibrary.NativeMessageController
-import com.sourcepoint.cmplibrary.SpClient
-import com.sourcepoint.cmplibrary.core.nativemessage.MessageStructure
-import com.sourcepoint.cmplibrary.model.ConsentAction
-import com.sourcepoint.cmplibrary.model.exposed.SPConsents
-import org.json.JSONObject
-
-class SpClientMock : SpClient {
```

Comment:
you decided to remove this class to mock it explicitly in each test suite, right?

Reply:
@bohdan-go-wombat this was used only in one test case. I refactor that test case to use a `@Mockk` instance instead.